okcurl/build.gradle

```diff
@@ -1,3 +1,8 @@
+plugins {
+  id "com.palantir.graal" version "0.3.0-6-g0b828af"
+  id "com.hpe.kraal" version "0.0.15"
```

Comment:
kraal fixes some incompatible kotlin bytecodes, may not be needed here without coroutines.